src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -569,161 +559,133 @@
+      "description": "Describes a single run of a static analysis tool, and contains the output of that run.",
       "additionalProperties": false,
       "type": "object",
       "properties": {
-        "invocation": 
-        {
-          "description": "A string that describes any parameterization for the tool invocation. For command line tools this string may consist of the completely specified command line used to invoke the tool.",
+        "tool": {
+          "description": "Information about the tool or tool pipeline that generated the results in this log. A results log can only contain results produced by a single tool or tool pipeline. A results log can aggregate results from multiple tool log files, as long as context around the tool run (tool command-line arguments and the like) is identical for all aggregated files.",
+          "$ref": "#/definitions/tool"
+        },
+
+        "invocation": {
+          "description": "A string containing the runtime parameters with which the tool was invoked. For command line tools, this string may consist of the completely specified command line used to invoke the tool.",
           "type": "string"
         },
 
-        "files": 
-        {
-          "description": "A dictionary each of whose keys is a URI and each of whose values is an array of fileReference objects representing the location of a single file target scanned during the run.",
+        "files": {
+          "description": "A dictionary each of whose keys is a URI and each of whose values is an array of file objects representing the location of a single file target scanned during the run.",
           "type": "object",
           "additionalProperties": {
             "type": "array",
             "items": {
-              "$ref": "#/definitions/fileReference"
+              "$ref": "#/definitions/fileData"
             }
           }
         },
 
-        "runStartTime": 
-        {
-          "description": "An optional string specifying the date and time at which the run started.",
+        "results": {
+          "description": "The set of results contained in an SARIF log. The results array can be omitted when a run is merely exporting rules metadata. It must be present (but may be empty) in the event that a log file represents an actual scan.",
+          "type": "array",
+          "minItems": 0,
+          "items": {
+            "$ref": "#/definitions/result"
+          }
+        },
+
+        "rules": {
+          "description": "An array of rule descriptor objects that describe all rules associated with an analysis tool or a specific run of an analysis tool.",
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/rule"
+          }
+        },
+
+        "startTime": {
+          "description": "A string specifying the date and time at which the run started.",
```

Comment:
1) Why don't we say 'Specifies the date and time at which the run started.' If we must constantly mention the type of the thing under discussion, why do we use this 'Specifies blah..' language elsewhere?
2) Would be nice to mention here that we have some precise ideas on the format string for the date and to provide a pointer to it if possible:

'A specially formatted string specifying the date and time... (refer to 6.1.4 of the SARIF spec for more information).

or some such